Find a functionally equivalent code:android.app.backup.BackupManager.requestRestore:COMMENT
Method Modifier: public      
Comment:/**
 * Restore the calling application from backup.  The data will be restored from the
 * current backup dataset if the application has stored data there, or from
 * the dataset used during the last full device setup operation if the current
 * backup dataset has no matching data.  If no backup data exists for this application
 * in either source, a nonzero value will be returned.
 *
 * <p>If this method returns zero (meaning success), the OS will attempt to retrieve
 * a backed-up dataset from the remote transport, instantiate the application's
 * backup agent, and pass the dataset to the agent's
 * {@link android.app.backup.BackupAgent#onRestore(BackupDataInput, int, android.os.ParcelFileDescriptor) onRestore()}
 * method.
 *
 * @param observer The {@link RestoreObserver} to receive callbacks during the restore
 * operation. This must not be null.
 *
 * @return Zero on success; nonzero on error.
 */

@@ -10,16 +10,16 @@
                 result = session.restorePackage(mContext.getPackageName(), observer);
             }
         } catch (RemoteException e) {
-            Log.w(TAG, "restoreSelf() unable to contact service");
+            Log.e(TAG, "restoreSelf() unable to contact service");
         } finally {
             if (session != null) {
                 session.endRestoreSession();

